HULKs/hulk
# You spin my head right round

## Introduced Changes

This PR refactors the CameraMatrix and projection implementations. We now have a `Camera` coordinate system that is consistent with literature, z being the camera front, y pointing down and x pointing right.
This allows for a concise camera projection using the intrinsic matrix.

Fixes #

## ToDo / Known Issues

*If this is a WIP describe which problems are to be fixed.*

## Ideas for Next Iterations (Not This PR)

* Nearly finished: projection of covariances in the image plane to the ground plane

## How to Test

* Upload, check twix image panel if balls are detected (at the correct positions) and all other projections
